src/frontend/src/lib/components/ui/ListItem.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import { type Snippet } from "svelte";
+  let {
+    children,
+    href,
+    hrefAriaLabel = "Link",
+    class: classes,
+  }: {
+    children: Snippet;
+    href?: string;
+    hrefAriaLabel?: string;
+    class?: string;
+  } = $props();
+</script>
+
+{#if href}
+  <a {href} class="contents h-full w-full" aria-label={hrefAriaLabel}>
```

Comment:
Could we put it inside the"li"? It's more semantically correct to put the "li" as a direct child of "ul".